src/a4/domain/Board.java

```diff
@@ -1,5 +1,71 @@
 package a4.domain;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Board {
+	List<BoardSpace> spaces = new ArrayList<BoardSpace>();
+
```

Comment:
We can limit it if you want, however adding/removing spaces is only done if we choose to do so. Even though our board is hard coded, I figure it is alright to allow it to be flexible if we need it to be.